.github/workflows/trivy.yaml

```diff
@@ -6,18 +6,35 @@ on:
     - cron: '0 10 * * *'
 
 jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      branches: ${{ steps.branches.outputs.branches }}
+    steps:
+      - name: Checking out repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Create branches to test
+        id: branches
+        run: |
+          BRANCHES=$(git branch -r | grep  -E '^  origin\/(((v|release-)?[0-9]+.[0-9]+)|(master|main))$' | \
+            sed -e 's#^  origin/##'  | jq -R -s -c 'split("\n")[:-1]')
```

Comment:
Can you write one or two example strings that pass validation with this regex?

Reply:
sure, for this repo they are
```
  origin/1.28
  origin/master
  origin/release-1.30
  origin/release-1.31
  origin/v1.1
```

Reply:
@eaudetcobello fixed